pom.xml

```diff
@@ -66,7 +66,7 @@
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <cdap.version>4.2.0-SNAPSHOT</cdap.version>
+    <cdap.version>4.1.1</cdap.version>
```

Comment:
this change is unintended?

Reply:
Hmm weird, not sure I'm just merging release into develop. This will change with the next PR anyways